spec/02-integration/09-hybrid_mode/11-status_spec.lua

```diff
@@ -107,8 +104,8 @@ for _, strategy in helpers.each_strategy() do
       end)
 
       -- now dp receive config from cp, so dp should be ready
-
-      skip_rpc_sync("should return 200 on data plane after configuring", function()
+      local it_no_rpc_sync = rpc_sync == "off" and it or pending
```

Comment:
could we rename it to `it_rpc_sync_off`?

Reply:
done